Fix Japanese language usage

olachinkei opened the pull request:
<!-- Thank you for your contribution! -->

<!-- Please add a reviewer to the assignee section when you create a PR. If you don't have the access to it, we will shortly find a reviewer and assign them to your PR. -->

## Why are these changes needed?

Stability AI asked Wandb to check some nuances of jp questions.


## Checks
What we changed.
https://docs.google.com/spreadsheets/d/1AT-Fsp-UH6T9vzOeBgfpPI3-Ti-JaoTSaIdHbtzhWe8/edit#gid=0

olachinkei committed:
modify some nuances

polm-stability commented on fastchat/llm_judge/data/japanese_mt_bench/question_full.jsonl:
```diff
@@ -1,14 +1,14 @@
 {"question_id": 1, "category": "coding", "turns": ["ディレクトリ内の全てのテキストファイルを読み込み、出現回数が最も多い上位5単語を返すPythonプログラムを開発してください。", "それを並列化（parallelize）することは可能ですか？"]}
 {"question_id": 2, "category": "coding", "turns": ["再帰（recursion）を用いてn番目のフィボナッチ数（Fibonacci number）を求めるC++プログラムを書いてください。", "今度は、各数が前の3つの数の和である数列を考えます。最初の3つの数は0、-1、-1です。n番目の数を求めるプログラムを書いてください。"]}
-{"question_id": 3, "category": "coding", "turns": ["HTMLでシンプルなウェブサイトを作成してください。ユーザーがボタンをクリックすると、4つのジョークからランダムに一つ表示されます。", "CSSを使ってジョークの色を赤にする方法は何ですか？"]}
+{"question_id": 3, "category": "coding", "turns": ["HTMLでシンプルなウェブサイトを作成してください。ユーザーがボタンをクリックすると、4つのジョークからランダムに一つ表示されるようにしてください。", "CSSを使ってジョークの色を赤にする方法を教えてください？"]}
```
```suggestion
{"question_id": 3, "category": "coding", "turns": ["HTMLでシンプルなウェブサイトを作成してください。ユーザーがボタンをクリックすると、4つのジョークからランダムに一つが表示されるようにしてください。", "CSSを使ってジョークの色を赤にする方法を教えてください。"]}
```

polm-stability reviewed (approved):
ありがとうございます！細かい点を修正しました。

polm-stability committed:
Update fastchat/llm_judge/data/japanese_mt_bench/question_full.jsonl

polm-stability committed:
Revise bus question

polm-stability committed:
Fix Socrates question

When this was translated, it seems like the first part of the question was accidentally replaced with the second half, so the question was just the second half twice.

shibuiwilliam reviewed (approved):
Nice catches!
